src/components/MAPI/apps/ClusterDetailWidgetApps.tsx

```diff
@@ -143,7 +209,7 @@ const ClusterDetailWidgetApps: React.FC<
     )
       return -1;
 
-    if (!upgradableApps) return undefined;
+    if (!upgradableApps) return 0;
```

Comment:
I think this means the upgrades available number in the apps widget will display `0` while we calculate this. Is this intentional?

Reply:
You are right, fixed